managed/ui/src/components/universes/UniverseForm/UniverseFormContainer.js

```diff
@@ -61,9 +61,14 @@ import { toast } from 'react-toastify';
 
 const mapDispatchToProps = (dispatch) => {
   return {
-    submitConfigureUniverse: (values) => {
+    submitConfigureUniverse: (values, uuid = null) => {
```

Comment:
`uuid = null` you can do it in the component only.

Reply:
This is used at multiple places, This extra param is not needed for most of them.